clang/docs/checkedc/Update-to-latest-LLVM-sources.md

```diff
@@ -1,81 +1,157 @@
 # Instructions for updating to the latest LLVM/Clang sources
 
-We are staying in sync with the LLVM/Clang mainline sources.   The baseline branch is a pristine copy of
-LLVM/Clang sources.  We periodically update the baseline branch and then push the changes to other branches.
-
-The first step is to create updated baseline branches:
-1. Create new branches of your local baseline branches (we suggest creating new
-branches so that you can run automated testing. You'll need new branches
-that you can push to GitHub).
-2. Update those branches to the latest sources.
-3. Run testing on those branches to make sure things are stable.
-
-The second step is to create updated master branches:
-  1. Create branches of your updated baseline branches.
-  2. Merge changes from the Checked C master branches into those branches.
-  3. Fix merge conflicts and run testing.  You will likely need to fix some issues
-     and re-run testing until all issues are fixed.
-
-The third step is to merge your changes back into your baseline and master branches.
-
-## Create updated branches of your baseline branches
-
-First create remote to the mirrored GitHub repo that contains the updated sources
-for LLVM/Clang. Go to your LLVM/Clang repo and do:
-
-    git remote add mirror https://github.com/llvm-mirror/llvm-project
-
-Then branch your baseline branch and merge changes into it:
-
+We are staying in sync with the LLVM/Clang mainline sources. The `baseline`
```

Comment:
I think we must clarify that the LLVM/Clang mainline sources are from upstream. Maybe instead of saying `LLVM/Clang` every time we can simply say `upstream`.